Cargo.toml

```diff
@@ -310,6 +313,7 @@ tungstenite = { version = "0.20.1", default-features = false }
 url = "2.4"
 utoipa = "4.2.0"
 utoipa-swagger-ui = "6.0.0"
+uuid = "*"
```

Comment:
why is this the only one with wildcard for a version?

Reply:
Good question, not sure. But I assume it was added for a good reason

Reply:
Also imho I'd prefer not to change it as part of this PR